docs/content/posts/2025-05-30-release-1.20.2.md

```diff
@@ -0,0 +1,133 @@
+---
+title:  "Apache Flink 1.20.2 Release Announcement"
+date: "2025-05-30T00:00:00.000Z"
+aliases:
+- /news/2025/05/30/release-1.20.2.html
+authors:
+- fcsaky:
+  name: "Ferenc Csaky"
+
+---
+
+The Apache Flink Community is pleased to announce the second bug fix release of the Flink 1.20 series.
+
+This release includes 25 bug fixes, vulnerability fixes, and minor improvements for Flink 1.20.
+Below you will find a list of all bugfixes and improvements (excluding improvements to the build infrastructure and build stability). For a complete list of all changes see:
+[JIRA](https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12315522&version=12355442).
+
+We highly recommend all users upgrade to Flink 1.20.2.
```

Comment:
Are we recommending this over version 2? I think we need a sentence positioning with version 2 here instead.

Reply:
may be we can use some generic phrase like ` or higher` otherwise it will pop up each release